Makefile

```diff
@@ -0,0 +1,60 @@
+# The release version.
+VERSION ?= v0.1
+
+# Etcd version. Should match the version in the database/Dockerfile.
+ETCD_VERSION ?= v3.3.7
+
+# Constants
+PYTHON_VERSION ?= 2.7
+
+calico/yaobank-customer:
+	docker build -t $@ customer
+
+calico/yaobank-summary:
+	docker build -t $@ summary
+
+calico/yaobank-database:
+	# Start an etcd server
+	-docker rm -f yaobank-etcd
+	docker run --detach \
+	  --net=host \
+	  --entrypoint=/usr/local/bin/etcd \
+	  --name yaobank-etcd quay.io/coreos/etcd:$(ETCD_VERSION) \
+	  --advertise-client-urls "http://$(LOCAL_IP_ENV):2379,http://127.0.0.1:2379,http://$(LOCAL_IP_ENV):4001,http://127.0.0.1:4001" \
+	  --listen-client-urls "http://0.0.0.0:2379,http://0.0.0.0:4001"
+	# Run the python population script
+	docker run --net=host -v $(CURDIR)/database:/database python:$(PYTHON_VERSION) sh -c "pip install python-etcd && cd /database && python loaddata.py"
+	# Copy out the default.etcd data from the etcd container and delete the etcd container
+	docker cp yaobank-etcd:/default.etcd $(CURDIR)/database
+	docker rm -f yaobank-etcd
+	# Create a packaged etcd with the populated data (this also modifies group/user settings for this data which is
+	# required for OpenShift).
+	docker build -t $@ database
+
+release:
+	$(MAKE) calico/yaobank-customer
+	$(MAKE) calico/yaobank-summary
+	$(MAKE) calico/yaobank-database
```

Comment:
I think the standard thing to do is to mark these targets as `.PHONY` and then they can be standard prereqs for the `release` target (which is probably also `.PHONY`)

Reply:
For some reason we seem to follow this pattern of calling MAKE for the release target in a number of our repos ... I didn't really question it tbh since I figured someone knew what they were doing :-)

Reply:
I think that might be some black magic related to the way we automatically compute the set of image targets. ¯\_(ツ)_/¯